packages/mdc-select/mdc-select.scss

```diff
@@ -211,8 +313,8 @@
     .mdc-list-divider {
       margin-left: -16px;
     }
+    // stylelint-enable
```

Comment:
Not sure why `plugin/selector-bem-pattern` was omitted here? We still include it in all other stylelint-enable comments.